integration-tests/main/src/main/resources/application.properties

```diff
@@ -18,10 +18,6 @@ quarkus.resteasy.gzip.max-input=10
 web-message=A message
 schedulerservice.cron.expr=0/10 * * * * ?
 
-quarkus.log.category.kafka.level=WARN
```

Comment:
This was a left-over from the earlier extraction of the tests into their own module.